subprojects/plugins/src/integTest/groovy/org/gradle/api/tasks/bundling/JarIntegrationTest.groovy

```diff
@@ -26,6 +27,7 @@ import java.util.jar.Manifest
 
 class JarIntegrationTest extends AbstractIntegrationSpec {
 
+    @TestReproducibleArchives
```

Comment:
Right, this class was different from the others in that you added `@TestReproducibleArchives` to many of the methods (I think all but one?).  I was thinking it would look cleaner to just make this one match the other archive tests and put the tests that don't need to be tested with reproducible jars into another test class.  WDYT?